src/xunit.performance.api/XunitPerformanceHarnessOptions.cs

```diff
@@ -15,6 +15,7 @@ internal sealed class XunitPerformanceHarnessOptions
         public XunitPerformanceHarnessOptions()
         {
             _outputDirectory = Directory.GetCurrentDirectory();
+            _temporaryDirectory = Path.GetTempPath();
```

Comment:
@brianrob I'm not sure we want to clean when the run completes; but perhaps scorch it when a new run starts? Cleaning it up automatically means throwing out stuff that is valuable when debugging issues/failures (as a scenario writer).

@guhuro either way, if we cleanup/scorch this, we need to make the default a subdirectory of `Path.GetTempPath()` and not the temp path itself--otherwise we will scorch stuff that other programs are using, and that would be Very Bad.